owncloudApp/src/main/java/com/owncloud/android/ui/activity/UploadFilesActivity.java

```diff
@@ -1,23 +1,22 @@
 /**
- *   ownCloud Android client application
- *
- *   @author David A. Velasco
- *   @author Shashvat Kedia
- *   @author Christian Schabesberger
- *   Copyright (C) 2018 ownCloud GmbH.
- *
- *   This program is free software: you can redistribute it and/or modify
- *   it under the terms of the GNU General Public License version 2,
- *   as published by the Free Software Foundation.
- *
- *   This program is distributed in the hope that it will be useful,
- *   but WITHOUT ANY WARRANTY; without even the implied warranty of
- *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- *   GNU General Public License for more details.
- *
- *   You should have received a copy of the GNU General Public License
- *   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ * ownCloud Android client application
  *
+ * @author David A. Velasco
+ * @author Shashvat Kedia
+ * @author Christian Schabesberger
+ * Copyright (C) 2018 ownCloud GmbH.
```

Comment:
Update this year to 2019

Reply:
Please avoid this on each touched file. So I made this 5 minute PR https://github.com/owncloud/android/pull/2444 to solve this at once.